docs/root/intro/version_history.rst

```diff
@@ -13,6 +13,18 @@ Version history
   now also defends against an HTTP/2 peer that does not open stream window once an entire response has been buffered to be sent to a downstream client.
 * listener: add runtime support for `per-listener limits <config_listeners_runtime>` on active/accepted connections.
 * overload management: add runtime support for :ref:`global limits <config_overload_manager>` on active/accepted connections.
+1.12.6 (July 7, 2020)
+=====================
```

Comment:
This file is such a mess due to `merge=union` rule merging conflicts. Could you clean it up (just replace with whatever is in my branch or upstream)? It doesn't matter too much, so feel free to ignore it.

Reply:
yes, i'll do that for all the PRs

Reply:
lame, sorry, github merged it before I could update the branch.  want a separate PR to update this file?